contracts/src/TokenRemote/TokenRemote.sol

```diff
@@ -205,6 +227,41 @@ abstract contract TokenRemote is ITokenRemote, TeleporterOwnerUpgradeable, SendR
         // Right-shift by 5 bits to divide by 32.
         return (payloadSize + 31) >> 5;
     }
+
+    function getIsCollateralized() public view returns (bool) {
```

Comment:
Thoughts on remove the "get" prefix on some/all of these public getters? `isCollateralized()` in particular seems fully descriptive already. Removing "get" may make them feel a bit closer to standard public state variable getters, but I don't have strong feelings either way.